### PR TITLE
Add set_runtime_environment functionality to nidigital

### PR DESCRIPTION
**Pull request by ni-jfitzger**
- [X] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/nimi-python/blob/master/CONTRIBUTING.md).
- [X] I've updated [CHANGELOG.md](https://github.com/ni/nimi-python/blob/master/CHANGELOG.md) if applicable.
- ~[] I've added tests applicable for this pull request~

This feature is tested through nifake unit tests.

### What does this Pull Request accomplish?
* nidigital metadata is updated from the latest internal export.
* `set_runtime_environment` call is added to nidigital/_library_interpreter.py

### List issues fixed by this Pull Request below, if any.
None

### What testing has been done?
None

### CODE REVIEW

File: docs/nidigital/class.rst [hunk 1]
```diff
@@ -1581,12 +1581,12 @@ get_channel_names
                 -   A range using a hyphen—for example, "0-3"
                 -   A range using a colon—for example, "0:3 "
 
-                You can combine comma-separated lists and ranges that use a hyphen or colon. Both out-of-order and repeated indices are supported ("2,3,0," "1,2,2,3"). White space characters, including spaces, tabs, feeds, and carriage returns, are allowed between characters. Ranges can be incrementing or decrementing.
+                You can combine comma-separated lists and ranges that use a hyphen or colon. Both out-of-order and repeated indices are supported ("2,3,0", "1,2,2,3"). White space characters, including spaces, tabs, feeds, and carriage returns, are allowed between characters. Ranges can be incrementing or decrementing.
```
Comment (marcoskirsch) [inline docs/nidigital/class.rst:1584]: weird

**Review by marcoskirsch — APPROVED** [review]

File: docs/nidigital/class.rst (hunk 1 above)
Comment (ni-jfitzger) [reply docs/nidigital/class.rst:1584]: Our Tech Writer insisted.

**Comment by ni-jfitzger** [comment]
@marcoskirsch the addition of this new method broke our unit tests, which want us to set its side effect. I'll have to fix it, before we can merge this.

File: docs/nidigital/class.rst (hunk 1 above)
Comment (ni-jfitzger) [reply docs/nidigital/class.rst:1584]: And looking at it now, I agree with him.

**Review by marcoskirsch — APPROVED** [review]

**Comment by ni-jfitzger** [comment]
The pending Read the Docs builds are builds that were cancelled because there were no changes. They should be showing as skipped or successful. Maybe the issue related to this isn't actually fixed (I don't want to reference it directly in each of our changes because each mention shows up in the issue) or maybe there's something not quite right with our configuration.

Either way, this is ready for merging.

**Pull request merged**